docs/deploy.md

```diff
@@ -1,108 +1,75 @@
 ## deploy task
-
-The `deploy` task supports deployment of one or more applications to the Liberty server.
+---
+The `deploy` task copies applications generated by the Gradle project to the Liberty server's `dropins` or `apps` directory. If an application is deployed while the server is running the task will verify that the application has started successfully.
 
 ### dependsOn
-`deploy` depends on `libertyStart` for a running server.
-
-### Properties
-
-See the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration) properties for common server configuration.
+`deploy` depends on any tasks of type `war` or `ear` so the package is ready before installation.  
+`deploy` also depends on `libertyCreate` to ensure that the server exists.
 
-The `deploy` task uses a `deploy` block inside the `liberty` block to define task specific behavior.
+### Parameters
 
-| Attribute | Type | Since | Description | Required |
-| --------- | ---- | ----- | ----------- | ---------|
-| file| String | 1.0 | Location of a single application to be deployed. The application type can be war, ear, rar, eba, zip, or jar. | Yes, only when a single file will be deployed. |
-| dir| String | 1.0 | Location of the directory where are the applications to be deployed.| Yes, only when multiples files will be deployed and `file` is not specified.|
-| include| String | 1.0 | Comma or space-separated list of patterns of files that must be included. All files are included when this attribute is omitted.| No |
-| exclude| String | 1.0 | Comma or space-separated list of patterns of files that must be excluded. No files are excluded when this attribute is omitted.| No |
+The `deploy` task can be configured using the `deploy` block. Below are the properties that can be set in the `deploy` block.
 
-### Examples
+| Attribute | Type  | Since | Description | Required |
+| --------- | ----- | ----- | ----------- | -------- |
+| apps | Object[] | 3.0 | Specifies which tasks/files should be deployed to the `apps` directory. Applications can be passed in as the task that generates the file or as an application file. | No |
+| dropins | Object[] | 3.0 | Specifies which tasks/files should be deployed to the `dropins` directory. Applications can be passed in as the task that generates the file or as an application file. | No |
 
-1. Deploys a single file.
-  ```groovy
-    apply plugin: 'liberty'
+In addition to the `deploy` block, the `stripVersion` and `looseApplication` properties are used for application installation. More information on these properties can be found in the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration).
 
-    liberty {
-        installDir = 'c:/wlp'
+Loose Application is only supported by Gradle 4.0 or higher for tasks of type ear.
```

Comment:
This can actually be misinterpreted the way it is written. I suggest "For tasks of type ear, loose application requires Gradle 4.0 or higher."